library/idr/__init__.py

```diff
@@ -0,0 +1,15 @@
+from connections import connection
+from connections import createHTTPsession
+
+from attributes import attributes_by_attributes
+from attributes import annotation_ids_by_field
+from attributes import get_phenotypes_for_gene
+
+from images import images_by_phenotype
+from images import simple_colocalisation
+
+from externalDBs import genes_of_interest_fromGO
```

Comment:
I think at this point the "experimental" (or "under development") methods _shouldn't_ be imported here.